src/functions/getValidatedCarNames.js

```diff
@@ -0,0 +1,19 @@
+import { PRINT_MESSAGES } from '../constants/messages.js';
+import { getUserInput } from '../utils/ioModule.js';
+import { splitByComma } from '../utils/stringUtils.js';
+import {
+  validateCarNames,
+  validateRacingCarInput,
+} from '../validator/validatePipeline.js';
+
+const getValidatedCarNames = async function getValidatedCarNamesFunc() {
+  const inputForRacingCars = await getUserInput(PRINT_MESSAGES.INPUT.CAR_NAME);
```

Comment:
`getValidatedCarNames` 함수의 인자로 `message`를 넘기는 방식으로 구현하면 어떨까요 ?
`PRINT_MESSAGES.INPUT.CAR_NAME` 에 의존하지 않고, 모듈간의 결합도를 낮출 수 있다고 생각해요.
```suggestion
const getValidatedCarNames = async function getValidatedCarNamesFunc(message) {
  const inputForRacingCars = await getUserInput(message);
```